src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -507,9 +506,11 @@ public struct DefaultStartConfiguration : IStartConfiguration
             /// Default Python script engine
             /// </summary>
             public string DefaultPythonEngine { get; set; }
+
             /// <summary>
             /// Disables ADP for the entire process for the lifetime of the process.
             /// </summary>
+            [Obsolete("This method will be removed in Dynamo 3.0 - please use Dynamo.Logging.Analytics.DisableAnalytics instead.")]
```

Comment:
So - this is still here, but it's not used anywhere - is that correct? If so maybe we should mention that it is effectively removed and is not to be used at all.